docs/pages/includes/determinetls.mdx

```diff
@@ -0,0 +1,26 @@
+<Details title="How to tell if TLS Routing is Enabled" opened={false}>
+
+<ScopedBlock scope={["oss", "enterprise"]}>
```

Comment:
Is the only difference between the `ScopedBlock`s the address of the cluster? We might consider using a `Var` for the address and instructing the user to fill it in.

Reply:
I think a Var is overkill for single use. I'm going to consolidate with a note to change the address as appropriate.